docs/UserGuide.md

```diff
@@ -18,85 +18,107 @@ title: User Guide
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 2. About
+## 2. How to use this User Guide
 
-### 2.1 Using this User Guide
+Welcome to *_MyInsuRec_ User Guide*!
 
-Welcome to *MyInsuRec User Guide*!
+Our team would like to thank you for choosing to use _MyInsuRec_! <br>
 
-This guide explains how you can use *MyInsuRec* to organize and manage your clients, meetings and products. This guide has been structured for users to quickly find and understand what they need.
+In order to smoothen the learning curve for you,
+this guide would allow you to effectively learn how to use _MyInsuRec_.
 
-Click on the links below to quickly get to what you need.
+Before we go on further, look out for these symbols in the User Guide as they provide some important message you might want to know!
 
-{ fill with instructions linking to different sections of the user guide }
+| Symbol               | Meaning                                                                                  |
+|----------------------|------------------------------------------------------------------------------------------|
+| :bulb:               | Tips and tricks to help you understand _MyInsuRec_ better.                               |
+| :information_source: | Additional information that can be helpful.                                              |
+| :exclamation:        | Crucial information that you should be aware of.                                         |
 
-e.g.
-1. xyz (link)
-2. 
+Now we can move on to talk more about _MyInsuRec_!
 
+<div markdown="block" class="alert alert-info">:exclamation:
+If you have **not installed** _MyInsuRec_, you can follow the [installation guide](#32-installation-instructions) here!
+</div>
+
+After you have installed _MyInsuRec_, you can
+1. Familiarise yourself with our [Terminologies](#23-terminologies) that are used throughout this User Guide.
+2. Understand the [layout of the interface](#24-user-interface) you will be interacting with.
+3. Follow this [step-by-step tutorial](#4-quick-start) to learn the basic features of _MyInsuRec_.
+4. Check out other advance features under [Features](#5-features) once you got the hang of the basic features.
+5. View our [glossary](#8-glossary) when you come across a unfamiliar term used.
+
+<div markdown="block" class="alert alert-info">:information_source:
+For **experienced user**, you can look at the [Command Summary](#7-command-summary) for a summarised table of all the commands available.
```

Comment:
`for the experienced user` or `for experienced users` sounds less awkward here imo

Reply:
also summarized for americanized